subprojects/docs/src/docs/userguide/workingWithFiles.adoc

```diff
@@ -229,29 +689,39 @@ Here is an example which maintains a copy of the project's runtime dependencies
+
+Creating an archive file is essentially a file copy, where the destination is implicit (the archive file itself). Here's an example:
 
 ++++
 <sample id="createZip" dir="userguide/files/archives" title="Creating a ZIP archive">
     <sourcefile file="build.gradle" snippet="zip"/>
 </sample>
 ++++
 
+TODO: Use a generic archive example. No need to use Java here.
+
 [TIP]
 .Why are you using the Java plugin?
 ====
 The Java plugin adds a number of default values for the archive tasks. You can use the archive tasks without using the Java plugin, if you like. You will need to provide values for some additional properties.
 ====
 
-The archive tasks all work exactly the same way as the `Copy` task, and implement the same `CopySpec` interface. As with the `Copy` task, you specify the input files using the `from()` method, and can optionally specify where they end up in the archive using the `into()` method. You can filter the contents of file, rename files, and all the other things you can do with a copy spec.
+The full power of copy specifications are available to you when creating archives, which means you can do content filtering, file renaming or anything else that is covered in the previous section. A particularly common requirement is to copy files into specific subdirectories in the archive, which can be achieved with `into()` <<Using child specifications,child specifications>>.
+
+Gradle does of course allow you create as many archive tasks as you want, but it's worth bearing in mind that many convention-based plugins provide their own. For example, the Java plugin adds a `jar` task for packaging a project's compiled classes in a JAR. Many of these plugins provide sensible conventions for the names of archives as well as the copy specifications used. We recommend you use these tasks wherever you can, rather than overriding them with your own.
 
 [[sec:archive_naming]]
 ==== Archive naming
 
-The format of `__projectName__-__version__.__type__` is used for generated archive file names. For example:
+Gradle has several conventions around the naming of archives and where they are created based on the plugins your project uses. The main convention is provided by the `base` plugin, which defaults to creating archives in the `$buildDir/distributions` directory and typically uses archive names of the form _[projectName]-[version].[type]_. The following example comes from a project named 'zipProject', hence the `myZip` task creates an archive named 'zipProject-1.0.zip':
```

Comment:
We mention the base plugin here a few times. Is there a good place we can link to where a user can learn more about what it is?

Reply:
- [ ] Link to [information on the Base Plugin](https://docs.gradle.org/current/userguide/standard_plugins.html#sec:base_plugins)